03LessonThree/in-class-project/daily-planet.html

```diff
@@ -4,7 +4,33 @@
         <title>The Daily Planet</title>
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
     </head>
+
     <body>
-        <!-- You code start here -->
+
+    <header>
+      <h1>Daily Planet</h1>
+      <nav>
+        <a href="http://espn.com">Sports</a> <br>
+        <a href="http://365thingsaustin.com">Local</a> <br>
+        <a href="http://www.mystatesman.com/news/business/experts-austins-economy-wont-falter-in-2016/npsp2/">Economy</a>
+      </nav>
+    </header>
+

```

Comment:
you're missing the <section> that the article is in. please put in :)  also, code is looking clean and organized! great work.